src/sordid-dragon.jquery.coffee

```diff
@@ -61,20 +61,32 @@ do ($=jQuery) ->
     # create a ghost under the cursor for us. That doesn't happen with touch
     # events, so we create one ourselves.
     $ghost = null
+    $ghostContainer = $("<div />")
+      .addClass("ghost-container")
+      .css
+        display: "none"
+        position: "fixed"
+        zIndex: 1000
     showGhost = (pageY) ->
+      $ghostContainer.appendTo("body").show()
+
       unless $ghost.is(":visible")
-        $ghost.addClass "sordidDragon-ghost"
-        $ghost.css
-          position: "fixed"
-          opacity: 1
-        $parent.append $ghost
+        $ghost
+          .addClass "sordidDragon-ghost"
+          .css
+            position: "fixed"
+            opacity: 1
+        $ghostContainer.append $ghost
+
       $ghost.css
         left: $placeholder.offset().left
         top: (pageY - ($placeholder.outerHeight() / 2)) - window.scrollY
         width: $placeholder.outerWidth() - window.scrollX
 
     hideGhost = ->
       $ghost?.remove()
+      $ghostContainer?.html("")
```

Comment:
Pragmatic @thoughtless is correct, though I would argue tooth and nail that 👻  *fixed in a transform element* are a serious concern for ![](https://camo.githubusercontent.com/427d0653f0cbb7b1d0db57a5cb98970bc0a7c408/687474703a2f2f7777772e63616d70666972656d617273686d616c6c6f77732e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031362f30352f67686f7374627573746572732d6f7665726c61792e706e67)